test/unit/solver/test_cuda_lu_decomposition.cpp

```diff
@@ -1,4 +1,4 @@
-#pragma once
+#include "test_lu_decomposition_policy.hpp"
```

Comment:
Why do we need to remove `pragma once` here and include a new header file?

Reply:
`pragma once` is only meant to be in header files. We are now sharing more code with the CPU side of things by including this header file, I think specifically something that checks results

Reply:
Ah, I see. Thanks for the clarification and it makes sense.